infer/models/java/Makefile

```diff
@@ -10,7 +10,7 @@ JACKSON_JAR = ../../../dependencies/java/jackson/jackson-2.2.3.jar
 REMOVE = rm -rf
 MAKE = mkdir
 
-CURRENT_DIR = $(shell pwd)
+CURRENT_DIR = $(shell printf "%q\n" "$(shell pwd)")
```

Comment:
Actually this is redundant with `CWD` above. Could you remove this line and replace the occurrences of `CURRENT_DIR` with `CWD` in the rest of this file?